contrib/debian/control

```diff
@@ -31,11 +32,14 @@ Package: dashd
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: peer-to-peer network based digital currency - daemon
- Dash is an experimental new digital currency that enables instant
- payments to anyone, anywhere in the world. Dash uses peer-to-peer
- technology to operate with no central authority: managing transactions
- and issuing money are carried out collectively by the network. Dash Core
- is the name of the open source software which enables the use of this currency.
+ Dash is a free open source peer-to-peer electronic cash system that
```

Comment:
I have no idea either. The one committed now is from Bitcoin 12.1 with Bitcoin replaced by Dash :)